modules/sievefilters/modules.php

```diff
@@ -223,7 +263,7 @@ public function process() {
             $blocked_wildcard = '@'.$domain;
             $new_blocked_list = [];
             foreach ($blocked_list as $idx => $blocked_sender) {
-                if (!mb_strstr($blocked_sender, $blocked_wildcard)) {
+                if (!strstr($blocked_sender, $blocked_wildcard)) {
```

Comment:
Is this intentional?

I am wondering because of https://github.com/cypht-org/cypht/issues/1224

Reply:
no it was not intentional. I understand, I correct